scripts/deploy.py

```diff
@@ -119,6 +120,7 @@ def main():
             token,
             gov,
             rewards,
+            gov,
```

Comment:
Do we want the script to include this as an option?

Reply:
Probably.

Reply:
updated on https://github.com/yearn/yearn-vaults/pull/271/commits/7e2de163b3bd9c8e49b6db8ee8ce733fadf2aed2